package/MDAnalysis/core/AtomGroup.py

```diff
@@ -5028,6 +5037,46 @@ def _matches_unpickling(self, anchor_name, n_atoms, fname, trajname):
         else:
             return False
 
+    def transfer_to_memory(self, frame_interval=1):
+        """Replace the current trajectory reader object with one of type
+        :class:`MDAnalysis.coordinates.memory.MemoryReader` to support in-place
+        editing of coordinates.
+
+        :Arguments:
+             *frame_interval*
+                 Read in every nth frame.
+
+        .. versionadded:: 0.15.0
+        """
+
+        from ..coordinates.memory import MemoryReader
+
+        if self.trajectory.format != "array":
+
+            # Try to extract coordinates using Timeseries object
+            # This is significantly faster, but only implemented for certain
+            # trajectory file formats
+            try:
+                coordinates = self.trajectory.timeseries(
+                    self.atoms, format='afc', skip=frame_interval)
```

Comment:
`skip` is deprecated and should be replaced by `step` (that works the exact same way)